Workbooks/Windows Virtual Desktop/Utilization Report/Utilization Report.workbook

```diff
@@ -716,10 +717,10 @@
                   "type": 3,
                   "content": {
                     "version": "KqlItem/1.0",
-                    "query": "let WVDHosts = dynamic([{WVDHosts}]);\r\nlet StartWin = startofday({TimeRange:start});    // date furthest in past\r\nlet EndWin = startofday({TimeRange:end}); \r\nlet lookbackWindow = 28d;  // 4 even weeks\r\nWVDConnections\r\n| extend Computername = strcat(\"{HostPool:label}\",\"/\", SessionHostName)\r\n| where Computername in (WVDHosts)\r\n| project CorrelationId, State, TimeGenerated, UserName\r\n| as Connections\r\n| where State == \"Started\" and TimeGenerated <= EndWin\r\n| extend StartTime = TimeGenerated\r\n| join kind=leftouter\r\n(\r\n    Connections\r\n    | where State == \"Completed\"\r\n    | extend EndTime = TimeGenerated\r\n)\r\non CorrelationId\r\n| extend EndTime = iff(isnull(EndTime), EndWin, EndTime)\r\n| where EndTime >= StartWin - lookbackWindow\r\n| project StartTime, EndTime, CorrelationId, UserName\r\n| extend TimeSlot = range(StartTime, EndTime, 1h)   //#3\r\n| mv-expand TimeSlot to typeof(datetime) limit 1000   //#4\r\n| evaluate sliding_window_counts(UserName, TimeSlot, StartWin, EndWin, lookbackWindow, 1d)\r\n| project Days = TimeSlot, RollingMAU = Dcount, Connections = Count\r\n",
+                    "query": "let WVDHosts = dynamic([{WVDHosts}]);\r\nlet StartWin = startofday({TimeRange:start});    // date furthest in past\r\nlet EndWin = startofday({TimeRange:end}); \r\nlet lookbackWindow = 28d;  // 4 even weeks\r\nWVDConnections\r\n| extend Computername = strcat(\"{HostPool:label}\",\"/\", SessionHostName)\r\n| where Computername in (WVDHosts)\r\n| project CorrelationId, State, TimeGenerated, UserName\r\n| as Connections\r\n| where State == \"Started\" and TimeGenerated <= EndWin\r\n| extend StartTime = TimeGenerated\r\n| join kind=leftouter\r\n(\r\n    Connections\r\n    | where State == \"Completed\"\r\n    | extend EndTime = TimeGenerated\r\n)\r\non CorrelationId\r\n| extend EndTime = iff(isnull(EndTime), EndWin, EndTime)\r\n| where EndTime >= StartWin - lookbackWindow\r\n| project StartTime, EndTime, CorrelationId, UserName\r\n| extend TimeSlot = range(StartTime, EndTime, 1h)   //#3\r\n| mv-expand TimeSlot to typeof(datetime) limit 1000   //#4\r\n| evaluate sliding_window_counts(UserName, TimeSlot, StartWin, EndWin, lookbackWindow, 1d)\r\n| project Days = TimeSlot, MAU = Dcount\r\n",
                     "size": 1,
                     "aggregation": 5,
-                    "title": "Monthly Users (MAU) and Connections",
+                    "title": "Monthly Average Users (MAU)",
```

Comment:
This should read `Monthly Active Users (MAU)`

Reply:
Should we go for "Monthly active users (MAU)"?

Reply:
You're right, that's the better capitalization!